### DISCUSSION TITLE
TOPRC 402mm FW190 protocol issues

### DISCUSSION THREAD

**Discussion by Pylonper** [pascallanger/DIY-Multiprotocol-TX-Module#999] [General]
Hello Pascal
First of all, thanks for your great work with all the mpm's protocols.

Maybe this is already known issue, but I could not find any information about it

I bought this TOPRC 402mm FW190 and have some issues setting this up. There is an addendum to the manual that states V761 TOPRC protocol should work.

It does bind, but only in low power mode and only with receiver no set to 0. But the main problem is that the servos are moving very sluggish with delays so it's not flyable.

The V761 with 4 channel works much better, the servos are moving fine and range is good, even though I had to bind also this in low power mode with receiver no 0. But the problem is that if there are more of these planes on the field they disturb each other, so we can only fly one at a time. I guess we all have the same transmitter ID?

Receiver is marked with HM-DD109R3, I tried both latest test build and also latest release, radiomaster t16s, taer.
I also tried binding with longer distance between model and transmitter to get it to bind in normal power mode without success

Please see pictures below

Does it support model match?
Also, is there only one transmitter id available?
Should it work with toprc protocol?
Really no issue that I have to use low power mode, just a bit odd..

Thanks for your great support

/Per

![FW190 box](https://github.com/user-attachments/assets/0b6bc2b7-5400-43e5-8d78-d8fc12fd9a2f)
![HM-DD109R3](https://github.com/user-attachments/assets/718f6969-b028-4639-9583-985ad5ae3c52)
![FW190 box2](https://github.com/user-attachments/assets/e53bdc0a-cf7e-4ff6-aa31-ebb3d47fdaf8)

**Comment by pascallanger** [comment]
Toprc planes work well with multi. 
If you have to bind using low power then it means your module is defective... There has been a production issue at radio master some long time back and I think your module's nrf24l01 is affected.

**Comment by Pylonper** [comment]
Hello Pascal

 

Thanks for the feedback, you were completely right.

Tried with my older external 4in1 module and it binds fine with toprc also on normal power.

 

Do you also know more in detail what the production issue was?

I’m quite used to work with electronics, so thinking if I can fix the internal module myself or if I need a new one

 

BR

 

/Per

**Reply by pascallanger** [reply]
I can't fully remember but I think it was a bad capacitor value around the nrf. I won't be capable to replace any component on the RF daughter board, everything is so small...

**Comment by Pylonper** [comment]
Ok, thanks for your reply.

Easier to just replace it then

 

BR

 

/Per
